sql/base/auth_database.sql

```diff
@@ -549,4 +547,4 @@ UNLOCK TABLES;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
 /*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
 
```

Comment:
The entry in the `updates` table is missing for your update `2015_09_23_15588_auth.sql`.